packages/mdc-base/foundation.js

```diff
@@ -19,21 +19,21 @@
  */
 export default class MDCFoundation {
 
-  /** @return enum{cssClasses} */
+  /** @return {!Object<string>} */
```

Comment:
I've tried multiple combinations and it seems the compiler ignores what follows `@return` when it is not wrapped with curly braces. `@return askjdlajs` passes but `@return {askjdlajs}` triggers the correct warning.

Then I'm not sure whether `@enum` can be used to annotate the return value of a method. This rightfully fails:

```
/**
 * @enum {string}
 */
this.strings = {
  foo: 'bar',
};
``` 

But this does not even though it's super wrong:

```
/**
 * @enum {string}
 */
getStrings() {
  return 123;
}
```

Whereas this throws a warning:

```
/**
 * @return {!Object<string>}
 */
getStrings() {
  return 123;
}
```

I didn't managed to get the object values type checked though. The code below should fail in theory.

```
/**
 * @return {!Object<string>}
 */
getStrings() {
  return {
    foo: 'bar',
    glagla: {},
  };
}
```